setup.py

```diff
@@ -276,9 +276,13 @@ def check_file_at_path(path2file):
                                 depends=['numba/_pymodule.h'],
                                 include_dirs=["numba"])
 
+    ext_string_conversion = Extension(name='string_conversion_ext',
```

Comment:
This is making a top-level module.  It should be nested inside numba.  i.e. `name='numba._string_conversion_ext'`.  But better yet is to build `numba/_string_conversion.c` into `numba._helperlib`.